### ISSUE TITLE
provide conda package

### ISSUE THREAD

**Issue by sanzoghenzo** [MSLNZ/msl-loadlib#22]
Hi, first of all thank you for this wonderful library!

is there any chance of having this package uploaded also to a conda repository, e.g [conda-forge](https://conda-forge.org/docs/maintainer/adding_pkgs.html)?

Thanks for the attention.

**Comment by jborbely** [comment]
Yes there is a chance. I have been considering this for a while, but it has been low in my priority queue. Knowing that others would find this useful is encouraging and helps to bump it up the queue.
